components/pagination/src/Pagination.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { FC, createElement as h } from 'react';
+import { StandardProps, classBuilder } from '@not-govuk/component-helpers';
+
+import '../assets/Pagination.scss';
+
+export type PaginationProps = StandardProps & {
+  results: number
+  resultsPerPage: number
+  page: number
+  previous?: string
+  pages?: number[]
+  next?: string
+};
+
+export const Pagination: FC<PaginationProps> = ({
+  children,
+  classBlock,
+  classModifiers,
+  className,
+  results,
+  resultsPerPage,
+  previous,
+  page,
+  pages,
+  next,
+  ...attrs
+}) => {
+  const classes = classBuilder(
+    'hods-pagination',
+    classBlock,
+    classModifiers,
+    className
+  );
+
+  var resultsFrom: number = ((page-1)*resultsPerPage)+1
+  var resultsTo: number = resultsFrom + (resultsPerPage - 1)
+  {resultsTo>results ? resultsTo=results : null}
```

Comment:
Please put spaces either side of binary operators. (I should really put in some linting rules or use prettier to sort this out.)

You probably don't need `: number` as that should be inferred anyway.

I don't think you should ever need to use `var` these days. Use either `let` or `const` and you should prefer `const`. You should be able to write this to use `const` but I can help if you get stuck.

Please use semi-colons. (Again I should add a linting rule for this.)

Reply:
In fact:
```js
const from = (page - 1) * resultsPerPage + 1;
const toTheoretical = page * resultsPerPage;
const to = (
  results < resultsToTheoretical
  ? results
  : resultsToTheoretical
);
```
Should work I think.
(I'm not sure why you had that `- 1` in there.)

Reply:
Thanks for this @daniel-ac-martin. I've incorporated the code above.